server/lib/nl/detection/heuristic_classifiers.py

```diff
@@ -376,3 +378,13 @@ def date(query_orig: str, ctr: Counters) -> Union[NLClassifier, None]:
   if attributes and attributes.dates:
     return NLClassifier(type=ClassificationType.DATE, attributes=attributes)
   return None
+
+
+def action(query_orig: str, ctr: Counters) -> Union[NLClassifier, None]:
+  verbs = dc.nl_detect_verbs(query_orig)
+  # When there are more than two verbs in a query, treat it as an action query.
+  if len(verbs) > 2:
+    return NLClassifier(
+        type=ClassificationType.ACTION,
```

Comment:
nit: how about calling it DETAILED_ACTION?

Reply:
Done